### PR TITLE
Split dashboard into widgets. Add build errors widget.

### PR DESCRIPTION
**Pull request by ss-gxp**
* Asynchronous loading of widgets
* Restrictions are minimal - any customization is available
* New widget to fix issue #120

### CODE REVIEW

**Comment by corpsee** [comment]
It is very impression feature :heart_eyes:! Thank you @ss-gxp! I will try to review and test this request today+tomorrow.

File: src/PHPCensor/Command/InstallCommand.php [hunk 1]
```diff
@@ -293,6 +293,14 @@ protected function getConfigInformation(InputInterface $input, OutputInterface $
                     ],
                 ],
             ],
+            'dashboard_widgets' => [
+                'all_projects' => [
+                    'side' => 'left',
+                ],
+                'history' => [
```
Comment (corpsee) [inline src/PHPCensor/Command/InstallCommand.php]: I guess it would be 'last_builds'?

Comment (ss-gxp) [reply src/PHPCensor/Command/InstallCommand.php]: Fix, thanks

File: src/PHPCensor/Controller/WidgetBuildErrorsController.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    */
+    public function index()
+    {
+        $builds = $this->buildStore->getAllProjectsLatestBuilds();
+
+        $this->view->builds = $builds['latest'];
+
+        $view = new b8\View('WidgetBuildErrors/update');
+        $view->builds = $builds['projects'];
+        $view->projects = $this->projectStore->getByIds(array_keys($builds['projects']));
```
Comment (corpsee) [inline src/PHPCensor/Controller/WidgetBuildErrorsController.php]: We need to filter archived projects somewhere here. Like in `WidgetAllProjectsController.php` (Line 107).

Comment (ss-gxp) [reply src/PHPCensor/Controller/WidgetBuildErrorsController.php]: Filter added (at contoller level, not store level, to avoid join)

Comment (corpsee) [reply src/PHPCensor/Controller/WidgetBuildErrorsController.php]: Thank you, controller level is ok.

**Comment by vinpel** [comment]
Dashboard visuasly empty on my isntallation 
 and got 2 javascript error (chrome) : 
````bash
Uncaught TypeError: Cannot set property 'allProjects' of undefined
    at all_projects.js:1
(anonymous) @ all_projects.js:1
last_builds.js:1 Uncaught TypeError: Cannot set property 'lastBuilds' of undefined
    at last_builds.js:1
(anonymous) @ last_builds.js:1
````

**Comment by corpsee** [comment]
Thank you @vinpel. @ss-gxp Can you try to see what is wrong?

**Comment by ss-gxp** [comment]
I think this browser cache. Try Ctrl-F5

**Comment by vinpel** [comment]
CTRL+F5 solve the problem !

**Pull request merged**